src/app/api/routing.py

```diff
@@ -54,3 +54,14 @@ async def create_user(user_table: Table, payload: UserAuth):
     pwd = await security.hash_password(payload.password)
     payload = UserCreation(username=payload.username, hashed_password=pwd, scopes=payload.scopes)
     return await create_entry(user_table, payload)
+
+
+async def update_user_pwd(user_table: Table, payload: UserCred, entry_id: int = Path(..., gt=0)):
```

Comment:
One should create another file to store those methods that do the interface between routing and crud. In my PR I added some lines in this file too. If this continue, the file will end up quite big.